### ISSUE TITLE
Agree with each Program on how they will continue operating post Program deprecation

### ISSUE THREAD

**Issue by mindthegab** [finos/finos-pmcs#31]
Per #22 on 4/22/2020 the Board has approved the following approach:

1. No new Programs will be created
2. Existing Programs can decide to continue operating under their current governance or disband as one/more top level Projects
3. FINOS Team can approve new top level Projects and manage their lifecycle transitions
4. The creation of a TAC (Technical Advisory Council) will be explored.

In this issue we'll track progress in working with each Program to agree on an approach to move forward on point 2. Attached below is an early recommendation that was discussed at the Board but that is by no means *binding* as we'll proceed to work with each Program to decide the approach they will take:

| STATE | PROGRAM  | PMC | OUTCOME | DECISION URL | NOTES / ACTIONS  |
|---|---|---|---|---|---|
| <ul><li>- [X] </li></ul>  | Data Tech  | @finos/dt-pmc  | Multiple Projects  | [Vote Result](https://groups.google.com/a/finos.org/forum/#!topic/data-tech-pmc/S76HfDFf7zY)  | [Make Project top Level and discontinue PMC Meetings](https://github.com/finos/finos-pmcs/issues/41)  |
| <ul><li>- [X] </li></ul>  | DEG  | @amberella @patrickmn | Disband  | [Vote Result](https://groups.google.com/a/finos.org/forum/#!topic/deg-pmc/J72udg_GJk8)   | [Nothing to do](https://github.com/finos/finos-pmcs/issues/35)  |
| <ul><li>- [X] </li></ul>  | DAV | @neilslinger  |  Multiple Projects | [Vote Result](https://groups.google.com/a/finos.org/forum/#!topic/dav-pmc/A1XL41bmt2M) |  [Make Projects top level](https://github.com/finos/finos-pmcs/issues/34), [Discontinue PMC meetings](https://github.com/finos/finos-pmcs/issues/33)  |
| <ul><li>- [X] </li></ul>  | Plexus  | @finos/plexus-pmc | Multiple Projects | [PMC Minutes](https://finosfoundation.atlassian.net/wiki/spaces/PLX/pages/1573453836/2020-04-24+Plexus+PMC+Meeting)  | [Make Projects Top Level](https://github.com/finos/finos-pmcs/issues/32), [Discontinue PMC Meetings](https://github.com/finos/finos-pmcs/issues/33) |
| <ul><li>- [X] </li></ul>  | FDC3  | @nkolba  | Continue as single project  | [PMC Decision minutes](https://finosfoundation.atlassian.net/wiki/spaces/FDC3/pages/1584464025/2020-05-01+Meeting)  | [Make Project Top Level and morph PMC](https://github.com/finos/finos-pmcs/issues/37)  |
| <ul><li>- [X] </li></ul>  | FDX  | @jbjonesjr  |  Multiple Projects and potentially SIGs | [Vote Issue](https://github.com/finos/fdx-pmc/issues/2)  | [Make Multiple Project](https://github.com/finos/fdx-pmc/issues/2) |
| <ul><li>- [X] </li></ul>  | FO  | @finos/fo-program  | Multiple Projects  | [Vote Result](https://groups.google.com/a/finos.org/forum/#!topic/fo-pmc/AS-5cpxViQQ)  | [Make Project top Level and discontinue PMC Meetings](https://github.com/finos/finos-pmcs/issues/42)  |
| <ul><li>- [X] </li></ul>  | VOICE | @tschady   |  Multiple Projects  |  [Vote Result](https://github.com/finos-voice/voice-program/issues/15)  | [Make Projects top level](https://github.com/finos/finos-pmcs/issues/34), [Discontinue PMC Meetings](https://github.com/finos/finos-pmcs/issues/33) |
| <ul><li>- [X] </li></ul>  | HADOUKEN |  @finos/hadouken-pmc   | Multiple Projects  |  [Vote Result](https://groups.google.com/a/finos.org/forum/#!topic/hadouken-pmc/NbRES3zdBkQ)  | [Make Projects Top Level](https://github.com/finos/finos-pmcs/issues/43) |
| <ul><li>- [X] </li></ul>  | SYMPHONY  | @finos/symphony-pmc   | Multiple Projects | [Vote Result](https://groups.google.com/a/finos.org/forum/#!topic/symphony-pmc/_qnCcDQp3xI)  | [Make Projects top level](https://github.com/finos/finos-pmcs/issues/38) |

<img width="1249" alt="Screen Shot 2020-04-24 at 5 45 00 PM" src="https://user-images.githubusercontent.com/961272/80267006-cd1f2780-8653-11ea-867c-f5ee79cc4a5e.png">

**Comment by ColinEberhardt** [comment]
For [Data Helix](https://github.com/finos/datahelix/), which lives under the Data Technologies program, we have a [roadmap](https://github.com/finos/datahelix/), an [organised backlog of work](https://github.com/finos/datahelix/projects) and [minuted meetings](https://finosfoundation.atlassian.net/wiki/spaces/DT/pages/1237811221/Steering+Committee+Minutes+for+Data+Helix+Project) - we'd be very happy to run this project independently and report directly. 

We're also [keen to transitions to active](https://github.com/finos/datahelix/pull/1674).

**Comment by mindthegab** [comment]
Thanks to all PMCs for quickly moving forward the conversation beyond Programs. 

As a summary of decisions: as detailed above all Programs have agreed to disband and to continue as separate Projects. In the case of FDX there's an open question whether groups like OSR or ODP should be a Special Interest Group (SIG) akin to what the LF uses. 

Next steps in this process, which we'll break down in separate issues are:

- **Closing the loop on Governance** as we removed Programs, we'll review and cleanup https://www.finos.org/governance to provide a simple, concise representation of our project and pan-project govenance

- **Wiki cleanup for PMC / Program information**: http://finosfoundation.atlassian.net/wiki/spaces/FINOS/

- **Removing the notion of programs from the website**: Mainly finos.org/programs and this will likely done contextually to publishing a new FINOS project landscape

- **Community Handbook cleanup**: Review, cleanup and simplify the [Community Handbook](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/80642059/Community+Handbook)

- **Exploring the creation of TAC / SIGs**: At our last Board meeting we discussed if one or more technical committees could be a useful construct.
